core-server/src/main/java/org/orbisgis/orbisserver/control/xml/GetCapabilitiesController.java

```diff
@@ -101,8 +103,8 @@ public Result displayXML(@Parameter("service") String service, @Parameter("versi
                  if(request.equals("GetCapabilities")){
                    try {
                      indexController.GetXMLFromGetCapabilities();
-                   } catch (JAXBException ignored) {
-                     LOGGER.error(I18N.tr("Unable to parse the incoming xml. \nCause : {0}.", ignored.toString()));
+                   } catch (JAXBException e) {
+                     LOGGER.error(I18N.tr("Unable to parse the incoming xml. \nCause : {0}.", e.getMessage()));
```

Comment:
I change the error message to "Unable to get the xml file corresponding to the GetCapabilities request."
I create a html error page with the exception and the text of the excpetion which is returned when the request fails, is that ok ?